client/protocols/collaborate.py

```diff
@@ -73,9 +73,8 @@ def run(self, messages):
             log.warning("Exception while waiting", exc_info=True)
 
     def start_firebase(self, messages):
-        access_token = auth.authenticate(self.assignment, force=False)
```

Comment:
I think the access_token is still necessary - it's passed in data. The server uses the auth token to login the user & check for permissions.